Wiki Reorg: PJS Ledger guides (#6141)
w3f/polkadot-wiki: docs/learn/learn-guides-ledger.md

[Polkadot Support Team](https://support.polkadot.network/support/home). For more user-friendly tools
see the [wallets](./wallets-index), [apps](./apps-index) and [dashboard](./dashboards-index) pages." />

The [Ledger devices](../general/ledger.md) are hardware wallets that keep your private key secured
on a physical device not directly exposed to your computer or the internet.

> The [Polkadot Generic application](../general/ledger.md#polkadot-generic-app) allows you to manage
> your tokens and other tokens on the Asset Hub and any other chain within the Polkadot ecosystem.

Reviewer: ```suggestion
The [Polkadot Generic application](../general/ledger.md#polkadot-generic-app) allows you to manage
your DOT/KSM on Polkadot/Kusama networks, tokens on their Asset Hubs and possibly all chains within the Polkadot ecosystem. It is versatile and capable of handling parachains and relay chains without being affected by their runtime upgrades. 
```

the address on your Ledger device. There are a few methods to check the balance of your Ledger
account. Check out
[this support article](https://support.polkadot.network/support/solutions/articles/65000169332-where-can-i-see-the-balance-of-my-account-)
for information.

## Navigating your Account

Once you have loaded your account on the “Accounts” tab, it should show a row with your Ledger
account. Your account’s DOT balance is on the row’s far right. Expanding the balance arrow will show
your balance details, such as locks or reserved amounts. For more information about the type of
balances, visit the [balances page](./learn-account-balances.md#balance-types-on-polkadot-js).

## Sending a Transfer with Ledger Devices

:::danger Verifying Extrinsics

Visit the
[**dedicated support page**](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-#Verify-an-extrinsic-using-Ledger)
and see [**this video tutorial**](https://youtu.be/bxMs-9fBtFk?t=360) tutorial to learn how to
verify extrinsics before signing them. The video will also mention potential attacks that can happen

> to you while signing transactions.

Reviewer: ```suggestion
while signing transactions.
```

:::

General instructions to send a transfer can be found on
[this support page](https://support.polkadot.network/support/solutions/articles/65000170304-how-to-send-transfer-funds-out-of-your-dot-account-on-the-polkadot-js-ui).
To sign transactions with your Ledger Nano check
[this support article](https://support.polkadot.network/support/solutions/articles/65000181994) or
see [this video tutorial](https://youtu.be/gbvrHzr4EDY?t=579).

## Receiving a Transfer

To receive a transfer on the accounts stored on your Ledger device, you must provide the sender
(i.e., the payer) with your address. To do so, follow the instructions on
[this support page](https://support.polkadot.network/support/solutions/articles/65000181866-how-to-receive-dot-to-my-account-on-polkadot-js-ui).

> :::caution Before giving anyone your address

Reviewer: ```suggestion
:::caution Sharing your account address
```

:::

The easiest way to get your address is to click on the account name. This will open a sidebar

> showing the information about your address and other information, such as on-chain identity. Another

Reviewer: ```suggestion
showing your address and other information, such as on-chain identity. Another
```

method is just clicking on your account's avatar icon - this immediately copies your address to the
clipboard.

:::note Your Asset Hub address is the same as your Relay Chain address

Make sure that you clarify to the sender that you wish to receive your tokens on the Asset Hub
parachain, otherwise (if you're receiving {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
tokens) they could be sent on the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
chain.

:::

## Staking

> For staking using Ledger devices, see

Reviewer: ```suggestion
For staking using Ledger devices, follow the instructions on
```
